modular-docs-manual/content/topics/introduction.adoc

```diff
@@ -2,3 +2,5 @@
 = Introduction
 
 This manual provides instructions on how to author modularly structured documentation based on user stories. The manual defines used terminology, describes components that form modular documentation, and instructs writers on how to use provided templates to turn user stories into modular documentation.
+
+IMPORTANT: As CCS strategic imperatives and Pantheon 2 development continue to evolve, the details and recommendations in this guide might be modified. The Flexible Customer Content (FCC) team recognizes that the fluctuation in recommendations can be frustrating for CCS teams trying to adopt modularization standards and apologizes in advance as the initiative pursues the best possible outcome for CCS documentation.
```

Comment:
See my comment about references to internal tooling (applies to teams as well): https://github.com/redhat-documentation/modular-docs/pull/121#discussion_r449559189